src/oc_erchef/habitat/config/chef_server_data_bootstrap.rb

```diff
@@ -0,0 +1,332 @@
+require 'fileutils'
```

Comment:
Part of me wonders if we can rewrite this as an escript which could (1) avoid adding more ruby gems into this hab package and (2) make it easy to share the same code between the old world and new.